tests/test_invites.py

```diff
@@ -90,6 +90,27 @@ def test_invited_email_address_obfuscation(self):
 
         self.assertEqual(redacted_address, "...@1...")
 
+        # Try using a username separator string
+        self.sydent.third_party_invite_username_separator_string = "-"
+
+        email_address = "johnathon-jingle-smithington@company-town.notarealtld"
+        redacted_address = store_invite_servlet.redact_email_address(email_address)
+        # Each individual component of the username should be obfuscated, but not the domain
+        self.assertEqual(redacted_address, "johnat...-jin...-smithi...@company-...")
+
+        # Try one with a separator at a word boundary
+        self.sydent.third_party_invite_username_separator_string = "."
```

Comment:
Not sure why you need to change the separator here? The tests would probably be clearer using `-` since `...` is the replacement.

Reply:
This was intended to check the four dots issue, but as we'll be solving that in another way this test can be changed to use `-` instead.